tests/playwright/pages/editor-page.js

```diff
@@ -1,18 +1,33 @@
 const { addElement, getElementSelector } = require( '../assets/elements-utils' );
-const { WpAdminPage } = require( './wp-admin-page' );
+const BasePage = require( './base-page.js' );
 
-exports.EditorPage = class EditorPage {
+module.exports = class EditorPage extends BasePage {
 	isPanelLoaded = false;
 
-	/**
-	 * @param {import('@playwright/test').Page} page
-	 */
-	constructor( page ) {
-		this.page = page;
+	constructor( page, testInfo ) {
+		super( page, testInfo );
+
+		this.previewFrame = this.page.frame( { name: 'elementor-preview-iframe' } );
 	}
 
-	getFrame() {
-		return this.page.frame( { name: 'elementor-preview-iframe' } );
+	async openNavigator() {
```

Comment:
Maybe make it a toggle?
```suggestion
	async toggleNavigator() {
```

Reply:
it is fine